### PR TITLE
Update transdecoder

### PR DESCRIPTION
**Pull request by stephanflemming**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

Related to: https://github.com/bioconda/bioconda-recipes/pull/26411

- improve tests
- add several plots as optional outputs
- some cleanups

### CODE REVIEW

File: tools/transdecoder/transdecoder.xml [hunk 1]
```diff
@@ -52,6 +53,7 @@ ${lo.S}
 #if 'log' in $oo.out
     |& tee '$out_log'
 #end if
+&& if ls 'output'/*'.pdf' >/dev/null 2>&1; then zip 'seqlogo.zip' 'output/'*'.pdf' -q -j; fi
```
Comment (abretaud) [inline tools/transdecoder/transdecoder.xml:56]: Are there many pdf files?

Comment (stephanflemming) [reply tools/transdecoder/transdecoder.xml:56]: Depends on the parameters. Everything I tried so far produced less than 10.

**Review by bgruening — APPROVED** [review]
ping @abretaud

**Comment by abretaud** [comment]
thanks @stephanflemming !

**Pull request merged**